templates/postgres.template.yml

```
@@ -167,36 +168,36 @@ run:
         - install -d -m 0755 -o postgres -g postgres /shared/postgres_backup
 
   - replace:
-      filename: "/etc/postgresql/12/main/postgresql.conf"
+      filename: "/etc/postgresql/13/main/postgresql.conf"
       from: /#?checkpoint_segments *=.*/
       to: "checkpoint_segments = $db_checkpoint_segments"
 
   - replace:
-      filename: "/etc/postgresql/12/main/postgresql.conf"
+      filename: "/etc/postgresql/13/main/postgresql.conf"
       from: /#?logging_collector *=.*/
       to: "logging_collector = $db_logging_collector"
 
   - replace:
-      filename: "/etc/postgresql/12/main/postgresql.conf"
+      filename: "/etc/postgresql/13/main/postgresql.conf"
       from: /#?log_min_duration_statement *=.*/
       to: "log_min_duration_statement = $db_log_min_duration_statement"
 
   - replace:
-      filename: "/etc/postgresql/12/main/pg_hba.conf"
+      filename: "/etc/postgresql/13/main/pg_hba.conf"
       from: /^#local +replication +postgres +peer$/
       to: "local replication postgres  peer"
 
   # allow all to connect in with md5 auth
   - replace:
-      filename: "/etc/postgresql/12/main/pg_hba.conf"
-      from: /^host.*all.*all.*127.*$/
+      filename: "/etc/postgresql/13/main/pg_hba.conf"
+      from: /^host.*all.*all.*137.*$/
```

Search-and-replace-all strikes again! 😄

I believe it should remain unchanged?

```suggestion
      from: /^host.*all.*all.*127.*$/
```

---

I thought I got all these! Good spotting.